components/content/src/file_info.rs

```diff
@@ -289,4 +301,26 @@ mod tests {
             Path::new("/home/vincent/code/site/content/posts/tutorials/python/index")
         );
     }
+
+    #[test]
+    fn correct_colocated_path() {
+        let files = vec![
+            FileInfo::new_page(
+                Path::new("/home/vincent/code/site/content/posts/tutorials/python/index.md"),
+                &PathBuf::new(),
+            ),
+            FileInfo::new_section(
+                Path::new("/home/vincent/code/site/content/posts/tutorials/_index.fr.md"),
+                &PathBuf::new(),
+            ),
+        ];
+
+        for file in files {
+            assert!(file.colocated_path.is_some());
+            if let Some(colocated_path) = file.colocated_path {
+                assert!(!colocated_path.starts_with('/'));
+                assert!(colocated_path.ends_with('/'));
```

Comment:
I just asserted what is really important for a colocated path which is not starting with a '/' and ending with a '/'.
I wanted to emphasize that because if you change the way you compute colocated path in the future but don't respect those two assertions, you'll break the current feature.

I definitely can split the check (one for the page and one for the section as they have different colocated path) in this test while adding a comment on what is really important to keep through time.
Would it be ok for you?

Reply:
Yes that would be fine